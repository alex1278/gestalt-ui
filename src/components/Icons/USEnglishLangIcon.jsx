import React from 'react';
import Isvg from 'react-inlinesvg';
import iconSVG from 'assets/icons/en-us.svg';

const USEnglishLangIcon = () => (
  <Isvg src={iconSVG} />
);

export default USEnglishLangIcon;
