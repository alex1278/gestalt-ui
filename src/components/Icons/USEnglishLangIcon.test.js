import React from 'react';
import { render } from 'react-testing-library';
import USEnglishLangIcon from './USEnglishLangIcon';


describe('(Component) USEnglishLangIcon', () => {
  it('renders component without exploding', () => {
    const { container } = render(<USEnglishLangIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
