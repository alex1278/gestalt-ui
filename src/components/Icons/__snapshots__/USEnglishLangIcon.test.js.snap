// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) USEnglishLangIcon renders component without exploding 1`] = `
<span
  class="isvg loading"
/>
`;
